Include/opcode.h

```diff
@@ -230,6 +230,9 @@ extern "C" {
 #define NB_INPLACE_TRUE_DIVIDE                  24
 #define NB_INPLACE_XOR                          25
 
+/* number of codewords for opcode+oparg(s) */
+#define OPSIZE 2
```

Comment:
I guess for now we're not contemplating the size depending on the opcode. Probably just as well.

Reply:
Yeah, it won’t be hard to change this macro if we decide to do that.